tipocket-ctl/tpctl/debug.py

```diff
@@ -0,0 +1,135 @@
+import inspect
+
+import click
+
+HELP_STRING = """
+generate debug environment into .env file.
+
+    Run `source .env` to get commands.
+
+\b
+You can find argument {deploy-id} and {cluster-namespace} in slack notification:
```

Comment:
we can get the `cluster-namespace` from the `deploy-id`.

Reply:
So we omit the possibility that namespace are specified in {namespace} flag?

Or maybe we should remove this flag and generate every namespace in tpctl.

Reply:
We can get the namespace from the argo workflow spec.

Reply:
Good idea.

Reply:
I parsed the output of `argo get -n argo $DEPLOY_ID -o json` with jq.